ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+import {
+  Box,
+  Container,
+  Select,
+  Heading,
+  Input,
+  Button,
+  VStack,
+  Image,
+  HStack,
+  Grid,
+} from '@chakra-ui/react';
+import { Text } from '@chakra-ui/layout';
+import { useState, Fragment } from 'react';
+
+import './styles.css';
+
+export default function HarvestGameInstructions() {
+  const gameType = [
+    {
+      game_num: 'Game #1',
+      name: 'SMUGGLING THROUGH SECURITY',
+      round: 3,
+      player: 5,
+      points: 45,
```

Comment:
Actually can we remove the points, I believe we cannot actually give this as a number since the scoring is based on an internal system right